### PR TITLE
Replace load paths with include and schema directives

### PR DESCRIPTION
**Pull request by ruippeixotog**
Replaces the load paths for defaults and schemas with `$include` and `$schema` directives that can be defined at any place in the config. The behavior of the new directives is best documented in `loader_spec.rb`.

### CODE REVIEW

File: spec/frise/loader_spec.rb [hunk 1]
```diff
@@ -75,28 +62,153 @@ def validators.short_string(_, str)
...
     ).to_stdout.and raise_error(SystemExit)
   end
 
   it 'should use an extra symbol table when provided' do
-    loader = Loader.new(
-      schema_load_paths: [fixture_path('_schemas')],
-      defaults_load_paths: [fixture_path('_defaults')]
+    loader = Loader.new(exit_on_fail: false)
+
+    conf = loader.load(fixture_path('loader_test2_all.yml'), '_id' => 'myobj')
```
Comment (belerophon) [inline spec/frise/loader_spec.rb]: Can these extra symbols be hashes or arrays?

File: spec/frise/loader_spec.rb [hunk 2]
```diff
@@ -75,28 +62,153 @@ def validators.short_string(_, str)
...
+    expect(conf).to eq(
+      'key1' => {
+        'value1_1' => 1,
+        'value1_2' => 'abc',
+        'key2' => { 'value2_1' => 2 }
+      }
+    )
+  end
+
+  it 'should process multiple inclusions and schemas in the same position' do
```
Comment (belerophon) [inline spec/frise/loader_spec.rb]: What should happen when the next include redefines some schema or default?

File: spec/frise/loader_spec.rb [hunk 3]
```diff
@@ -75,28 +62,153 @@ def validators.short_string(_, str)
       str.is_a?(String) && str.length < 5 || (raise "expected a short string, found #{str.inspect}")
     end
 
-    loader = Loader.new(
-      schema_load_paths: [fixture_path('_schemas')],
-      validators: validators
-    )
+    loader = Loader.new(validators: validators, exit_on_fail: true)
```
[the remark this answers is not in the document]
Comment (ruippeixotog) [reply spec/frise/loader_spec.rb]: I don't think it is, these are Ruby keyword arguments.

File: spec/frise/loader_spec.rb (hunk 2 above)
Comment (ruippeixotog) [reply spec/frise/loader_spec.rb]: The includes are merged in order, so the original file has priority over the first include and the first include has priority over the second. That's tested in this spec.

File: spec/frise/loader_spec.rb (hunk 1 above)
Comment (ruippeixotog) [reply spec/frise/loader_spec.rb]: They can, I added in 48a81d6 an object here to test that.

File: lib/frise/loader.rb [hunk 4]
```diff
@@ -10,53 +11,120 @@ module Frise
   # The load method loads a configuration file, merges it with the applicable defaults and validates
```
[the remark this answers is not in the document]
Comment (ruippeixotog) [reply lib/frise/loader.rb]: Changed slightly in 48a81d6.

**Pull request merged**